portfolio/src/main/webapp/style.css

```diff
@@ -36,6 +36,15 @@ a{
   float:right;
 }
 
+#dynamicDescription{
+  font-size: x-large;
+  font-family:Courier New;
```

Comment:
nit: think the spacing after the colon `:` should be consistent here so

`font-family: Courier New;`

Reply:
I'll fix that and push the update to the same branch